templates/org/settings/navbar.tmpl

```diff
@@ -7,6 +7,9 @@
 		<a class="{{if .PageIsSettingsHooks}}active{{end}} item" href="{{.OrgLink}}/settings/hooks">
 			{{.i18n.Tr "repo.settings.hooks"}}
 		</a>
+		<a class="{{if .PageIsSettingsHooks}}active{{end}} item" href="{{.OrgLink}}/settings/labels">
```

Comment:
```suggestion
		<a class="{{if .PageIsOrgSettingsLabels}}active{{end}} item" href="{{.OrgLink}}/settings/labels">
```

I might be wrong, but it's likely not hooks 🙂

Reply:
one of 50 copy/paste errors, fixed thanks